src/layman/layer/rest_workspace_test.py

```diff
@@ -63,7 +63,7 @@
 }
 """
 
-num_layers_before_test = 0  # pylint: disable=invalid-name
+num_of_publications = PublicationCounter()
```

Comment:
`num_of_publications` -> `PUBLICATION_COUNTER` (not sure, why pylint accepts it in lowercase)

Reply:
I think, that uppercase is for constants and this is not constant. It is a global variable, so I used lowercase.